Cargo.toml

```diff
@@ -19,6 +19,9 @@ exclude = ["./github"]
 [badges]
 codecov = { repository = "petabi/petal-decomposition", service = "github" }
 
+[lints.clippy]
+pedantic = "warn"
```

Comment:
The Clippy warnings in lapack.rs are now suppressed directly in the code, so adjustments in `[lints.clippy]` may not be necessary unless there’s a specific reason. Could you clarify if these changes address any additional issues?

Reply:
I noticed that [a check](https://github.com/petabi/petal-decomposition/blob/6904f0c02c64adae0596097b3af6e4a07687c721/.github/workflows/ci.yml#L52) was running that explicitly checked for pedantic lints. I didn't know about this check, so running Clippy before opening the PR did not reveal that the lapack.rs pedantic violations were going to be a problem. The goal of this line is to make it apparent to contributors when they run Clippy that they need to address pedantic lints. I can remove it if you wish.

Reply:
Come to think of it, `cargo clippy -- -D warnings -W clippy::pedantic` could now just become `cargo clippy -- -D warnings`. Let me know if you'd like me to also make that change if you don't want to remove this.